src/fondant/cli.py

```diff
@@ -390,9 +394,37 @@ def register_compile(parent_parser):
         default="vertex_pipeline.yml",
     )
 
+    # sagemaker parser
+    sagemaker_parser.add_argument(
+        "ref",
+        help="""Reference to the pipeline to run, can be a path to a spec file or
```

Comment:
I see that this is the same for the other frameworks, but this is probably the help text that should be on the runner. For the compiler, this always needs to be a module containing the pipeline instance I guess?